lib/open_api_spex/cast/discriminator.ex

```diff
@@ -37,7 +37,7 @@ defmodule OpenApiSpex.Cast.Discriminator do
     end
   end
 
-  defp cast_discriminator(%_{value: %{} = value, schema: schema} = ctx) do
```

Comment:
Can you please add a test for this change?

Reply:
Do you mean for this specific line or for the whole change? I was pretty sure to have committed a test but there is not, I'll do later

Reply:
@zorbash test added